### PR TITLE
Update the LICENSE

### PR DESCRIPTION
**Pull request by lilin90**
### What is changed, added or deleted? (Required)

Update the `LICENSE` to [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/)

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/13045
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,3 +36,7 @@
 ## 贡献者数量增长图
 
 [![Contributor over time](https://contributor-graph-api.apiseven.com/contributors-svg?chart=contributorOverTime&repo=pingcap/docs-cn)](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=pingcap/docs-cn)
+
+## License
+
+所有文档均遵循 [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/) 条款。
```
Comment (lilin90) [inline README.md:42]:
```suggestion
所有文档的许可证均为 [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/)。
```

File: README.md [hunk 2]
```diff
@@ -36,3 +36,7 @@
 ## 贡献者数量增长图
 
 [![Contributor over time](https://contributor-graph-api.apiseven.com/contributors-svg?chart=contributorOverTime&repo=pingcap/docs-cn)](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=pingcap/docs-cn)
+
+## License
+
+所有文档的许可证均为 [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/)。
```
Comment (ran-huang) [inline README.md:42]: 这里是指所有版本的文档吗？

Comment (lilin90) [reply README.md:42]: v7.0 开始的文档，我来调整下

Comment (lilin90) [inline README.md:42]:
```suggestion
自 TiDB v7.0 起的所有文档的许可证均为 [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/)。
```

File: README.md [hunk 3]
```diff
@@ -36,3 +36,7 @@
 ## 贡献者数量增长图
 
 [![Contributor over time](https://contributor-graph-api.apiseven.com/contributors-svg?chart=contributorOverTime&repo=pingcap/docs-cn)](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=pingcap/docs-cn)
+
+## License
+
+自 TiDB v7.0 起的所有文档的许可证均为 [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/)。
```
Comment (lilin90) [inline README.md:42]:
```suggestion
自 TiDB v7.0 起，所有文档的许可证均为 [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/)。
```

**Pull request merged**